src/es/components/molecules/multiLevelNavigation/MultiLevelNavigation.js

```diff
@@ -216,6 +222,7 @@ export default class MultiLevelNavigation extends Mutation() {
   }
 
   disconnectedCallback () {
+    self.removeEventListener('blur', this.noScroll)
```

Comment:
You have to loop over all preventDefaultInputSearch fields and remove the event listener on them. Same as you add, you remove.